ports/kf5config/portfile.cmake

```
@@ -0,0 +1,55 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO KDE/kconfig
+    REF v5.64.0
+    SHA512 b8919e552a35ed3f4882d2d15205c5576be985e9f6e4e834d26587c9288e85a2ab27504a1531d1af70e8db4dc7ed71a6b0caf91c7310dace81177b68aa6a97e5
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS 
+        -DBUILD_HTML_DOCS=OFF
+        -DBUILD_MAN_DOCS=OFF
+        -DBUILD_QTHELP_DOCS=OFF
+        -DBUILD_TESTING=OFF
+)
+
+vcpkg_install_cmake()
+
+file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/${PORT})
+
+if(VCPKG_TARGET_IS_WINDOWS OR VCPKG_TARGET_IS_UWP)
+    file(RENAME ${CURRENT_PACKAGES_DIR}/bin/kconfig_compiler_kf5.exe ${CURRENT_PACKAGES_DIR}/tools/${PORT}/kconfig_compiler_kf5.exe)
+    file(RENAME ${CURRENT_PACKAGES_DIR}/bin/kconf_update.exe ${CURRENT_PACKAGES_DIR}/tools/${PORT}/kconf_update.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/bin/kreadconfig5.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/bin/kwriteconfig5.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kconfig_compiler_kf5.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kconf_update.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kreadconfig5.exe)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kwriteconfig5.exe)
+else()
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/bin/kreadconfig5)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/bin/kwriteconfig5)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kreadconfig5)
+    file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/kwriteconfig5)
+endif()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/KF5Config)
+
+vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/${PORT})
+file(APPEND ${CURRENT_PACKAGES_DIR}/tools/${PORT}/qt.conf "Data = ${VCPKG_ROOT_DIR}/installed/${TARGET_TRIPLET}/data")
+vcpkg_copy_pdbs()
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static" OR VCPKG_TARGET_IS_LINUX)	

```

```suggestion
if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")	
```
`VCPKG_TARGET_IS_LINUX` seems to be not needed here. Since on Linux, `VCPKG_LIBRARY_LINKAGE` is set as `static` in x64-linux triplet.